### ISSUE TITLE
Invalid warning on Worker assignment

### ISSUE THREAD

**Issue by jrieken** [google/tsec#22]
* use vscode source
* run `yarn tsec-compile-check`
* apart from a few other errors this being reported

```
src/vs/workbench/services/extensions/worker/polyfillNestedWorker.ts:47:8 - error TS21228: [ban-worker-calls] Constructing Web Workers can cause code to be loaded from an untrusted URL.

47  	self.Worker = <any>class { constructor() { throw new TypeError('Nested workers from within nested worker are NOT supported.'); } };
    	     ~~~~~~
````

**Comment by jrieken** [comment]
related https://github.com/microsoft/vscode/issues/103699

**Comment by uraj** [comment]
The patch will be available in v0.1.2.

**Comment by uraj** [comment]
This is only partially fixed by 5c838d8. Reopen.
